src/components/WorkshopPage/WorkshopPage.js

```diff
@@ -54,6 +55,10 @@ const useStyles = makeStyles(theme => ({
 		color: theme.palette.primary.main,
 		fontFamily: theme.typography.fontFamily
 	},
+	note: {
```

Comment:
you can delete this `note` section because we aren't using these styles anywhere